dexterity/membrane/behavior/password.py

```diff
@@ -22,6 +22,23 @@
 from zope.interface import provider
 
 
+class BCRYPTEncryptionScheme(object):
+    """A BCRYPT AuthEncoding."""
+
+    def encrypt(self, pw):
+        return bcrypt.hashpw(pw, bcrypt.gensalt())
+
+    def validate(self, reference, attempt):
+        try:
+            valid = bcrypt.hashpw(attempt, reference) == reference
+        except ValueError:
+            valid = False
+        return valid
+
+
+AuthEncoding.registerScheme('BCRYPT', BCRYPTEncryptionScheme())
```

Comment:
For bonus points, this should probably be added to AccessControl itself, and be made the default there.

I would say, add a condition here: `if 'BCRYPT' not in AuthEncoding.listSchemes(): ...`. Looks like it will work fine without this condition, but you may end up with two BCRYPT schemes if it ever gets added to the core.